data/hai/hai reveal 2 start.txt

```diff
@@ -371,7 +371,7 @@ mission "Hai Rescue: Prison Role-Players"
 				`	(Ignore it and come back later.)`
 					defer
 			`	You put your ship down in a forest clearing and step out to find some wary-looking Hai with a directional transmitter and a very broken freighter in pirate colors.`
-			`	One of them, a lady, steps forward. "Captain <last>! Please help us. We need to get home to <destination>."`
+			`	One of them, a lady, steps forward. "Oh, it's Captain <last>! Please help us. We need to get home to <destination>."`
```

Comment:
When I saw this scene while playing, I thought it was a trap and the pirates had noticed me trying to save all these Hai, because the conversation seemed too uncanny to be true. I still saved those Hai in the end, but I was sure I was going to see pirates jump out and kill me. I added this bit in to help get rid of this feeling. 

Also, imagine the feeling of relief when that stranded Hai sees someone she recognises instead of some other random merchant.

Reply:
How about this? 
> One of them, a lady, steps forward, and as she does so her face lights up as if she's seen you before. "Captain <last>, Captain <last>! Please help us, we need to get home to <destination>!"

Reply:
I like that.